lib/ansible/plugins/callback/default.py

```diff
@@ -304,7 +316,7 @@ def v2_playbook_on_stats(self, stats):
         self._display.display("", screen_only=True)
 
         # print custom stats
-        if self._plugin_options.get('show_custom_stats', DEFAULT_SHOW_CUSTOM_STATS) and stats.custom:
+        if self.show_custom_stats:
```

Comment:
Is the `and stats.custom` part of the condition not actually needed?

Reply:
nope, that was overzealous deletion